.github/workflows/reusable-unit.yml

```diff
@@ -77,11 +78,11 @@ jobs:
         run: yarn ui build
 
       - name: Build react-core package
-        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' }}
+        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-geo' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' }}
         run: yarn react-core build
 
       - name: Build react package
-        if: ${{ matrix.package == 'react-storage' || matrix.package == 'react-liveness' && matrix.package != 'react'}}
+        if: ${{ matrix.package == 'react-geo' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' && matrix.package != 'react' }}
```

Comment:
Do we really need  `matrix.package != 'react'` here?

Reply:
This step was to only build `ui-react` for `ui-react-{category}` packages.
Next step which is
```
- name: Build ${{ matrix.package }} package
  if: ${{ matrix.package != 'ui' }}
  run: yarn ${{ matrix.package }} build
```
builds all packages. So, without `matrix.package != 'react'`, we would be building `ui-react` twice.

Reply:
Looking at this code, I think we just need the `||`s, because if 

```
matrix.package == 'react-geo' || matrix.package == 'react-storage' || matrix.package == 'react-liveness'
```

is `true`, then `matrix.package != 'react'` for granted.

Reply:
Oh yeah, that's true. Removed `&& matrix.package != 'react'`